website/docs/d/ram_resource_share.html.markdown

```diff
@@ -13,6 +13,7 @@ description: |-
 ## Example Usage
 ```hcl
 data "aws_ram_resource_share" "example" {
+  resource_owner = "SELF"
```

Comment:
Good catch! Could you move `resource_owner = "SELF"` down a line so that it comes after the `name` attribute to keep it consistent with the filters example.

Reply:
Done.